### PR TITLE
Addition of a new canonical Chiller.

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

**Review by tasodorff — APPROVED** [review]

**Comment by cstirdivant** [comment]
@tasodorff is this redundant with the existence of CH_SS_SWTC? If it truly is an air-cooled chiller no need to use `SCHWTC`

**Comment by ghairapetian** [comment]
> @tasodorff is this redundant with the existence of CH_SS_SWTC? If it truly is an air-cooled chiller no need to use `SCHWTC`

It's truly an air cooled chiller. However, SWTC do not specify a cooling_percentage_command. I can add it as an optional field if you confirm.

**Comment by mschulze17** [comment]
@tasodorff can you PTAL?

**Comment by tasodorff** [comment]
> 

Please add it as an optional field to SWTC and then you may proceed with using the existing `CH_SS_SWTC` type.

File: ontology/yaml/resources/HVAC/entity_types/CH.yaml [hunk 1]
```diff
@@ -25,6 +25,14 @@ CH_SS_SWTC:
   - SS
   - SWTC
 
+CH_SS_SCHWTC:
+  description: " Simple air cooled chiller."
+  is_canonical: true
+  implements:
+  - CH
+  - SS
+  - SCHWTC
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/CH.yaml:34]: Add the field you need (`cooling_percentage_command`) as an optional field on `SWTC` and reuse existing type `CH_SS_SWTC`.

**Review by tasodorff — DISMISSED** [review]

**Comment by charbull** [comment]
@ghairapetian friendly ping

**Comment by ghairapetian** [comment]
@charbull , I made the correction requested. If it's not enough or wrong, do not hesitate.

**Pull request merged**